src/XMakeBuildEngine/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -445,6 +445,8 @@ public void BeginBuild(BuildParameters parameters)
                     throw _threadException;
                 }
 
+                loggingService.LogTelemetry("Event from build manager", null);
```

Comment:
>loggingService.LogTelemetry("Event from build manager", null); [](start = 16, length = 62)

This is just an example of how to log within MSBuild itself.  This will be removed before merging.